routing_model/src/actions.rs

```diff
@@ -177,6 +179,12 @@ impl Action {
             .set_offline_state(&Name(node.0.name));
     }
 
+    pub fn set_node_back_online_state(&self, node: Node) {
+        self.0
+            .borrow_mut()
+            .set_online_state(&Name(node.0.name));
```

Comment:
I guess this should also call the set_relocating to match the diagram I linked.

Reply:
Ah yes good catch!